tox.ini

```diff
@@ -67,6 +67,8 @@ changedir =
 
 commands_pre =
   test: pip install .
+  test: bash -c '[[ -e {envbindir}/mock_server-x64-linux ]] || curl -L -o {envbindir}/mock_server-x64-linux https://github.com/googleinterns/cloud-operations-api-mock/raw/master/cmd/mock_server-x64-linux'
```

Comment:
@aabmass Regarding you previous comment about not needing the `bash -c`, it still works without it, however I get a bunch of warnings: `WARNING: test command found but not installed in testenv. cmd: /usr/bin/chmod. DEPRECATION WARNING: this will be an error in tox 4 and above!`